clang/lib/CodeGen/CGObjCMac.cpp

```diff
@@ -4856,7 +4773,7 @@ void CGObjCMac::EmitTryOrSynchronizedStmt(CodeGen::CodeGenFunction &CGF,
     if (PropagatingExnVar.isValid()) {
       PropagatingExn = CGF.Builder.CreateLoad(PropagatingExnVar);
 
-    // Otherwise, just look in the buffer for the exception to throw.
+      // Otherwise, just look in the buffer for the exception to throw.
```

Comment:
This comment continues the narrative of the comment preceding the `if` and should be indented at the same level. Indenting a comment in this position makes it appear to be a conclusion of a narrative within the `if` block, which is also a reasonable thing to have, but is incorrect in this case. `clang-format` should not be reindenting comments that immediately precede an `else` if it doesn't understand this distinction (which of course it cannot).

At any rate, please leave this at its current indentation, and please review the rest of your patch to make sure clang-format hasn't made the same mistake somewhere else.

Reply:
Found another two cases of similar situation, adjusted to make clang-format happy and semantically meaningful.

Reply:
I've filed https://github.com/llvm/llvm-project/issues/126844 with clang-format. I know fighting the tool is a pain in the meantime, but please just manually restore the original formatting for these comments before committing. Otherwise, I trust you and this LGTM.